model/models.py

```diff
@@ -6,8 +6,7 @@
 from utils.losses import weighted_sum_losses
```

Comment:
Why is that separate import? Can we not all import it once because below you import from utils.losses again.